platform/FileHandle.h

```diff
@@ -114,10 +114,10 @@ class FileHandle {
      *
      *  @return         Size of the file in bytes
      */
-    virtual size_t size()
+    virtual off_t size()
```

Comment:
Minor. Changes behaviour if an error is encountered during file size operation, but currently that can not happen in the FAT filesystem. Changes behaviour if a file exceeds 2GB, but that also can't happen in the FAT filesystem.

This is already planned for 5.5 so even if the user does something crazy like take a member function pointer to this guy they should expect a change.

I was considering splitting out the `off_t` into seperate pr. Would that help visibility of the change to users?